aws-rds-globalcluster/aws-rds-globalcluster.json

```diff
@@ -59,8 +59,7 @@
     "/properties/GlobalClusterIdentifier",
     "/properties/SourceDBClusterIdentifier",
     "/properties/StorageEncrypted",
-    "/properties/Engine",
-    "/properties/EngineVersion"
```

Comment:
by removing this you are implying a significant change in resource behavior - now customer should not receive any interruptions. My questions is about what the experience should be. Engine version upgrade should cause no downtime but what about downgrade. Will customer be able to successfully provision lower version? If not then, we should think about preserving replacement behavior on version downgrade. One way of doing so is to mark `EngineVersion` as `conditionalCreateOnly`. This would imply that resource will decide whether to replace during the runtime

Reply:
Good point but i think this behaviour is consistent with regular cluster update requests, the modify cluster request should just fail without replacement.

Reply:
so we are ok making this feature a forward only update?

Reply:
Correct me if I'm wrong but I think the engine version upgrade has been a forward-only thing for other resource types as well. This change just makes it consistent with other types.

Compared with DbCluster type: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/blob/37268603fa31226859ed4cf3af9511bce4ae294d/aws-rds-dbcluster/aws-rds-dbcluster.json#L383-L402

https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-rds/blob/37268603fa31226859ed4cf3af9511bce4ae294d/aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/Translator.java#L223-L226